config/routes.yaml

```diff
@@ -6,4 +6,8 @@ controllers:
 
 curl_search:
     path: /books/{title}
-    controller: App\Controller\BookController::searchBooksByTitle
\ No newline at end of file
+    controller: App\Controller\BookController::searchBooksByTitle
+
+curl_search:
+    path: /books/{category}
+    controller: App\Controller\BookController::searchBooksByCategory
```

Comment:
change path to /books?category={category}

Reply:
done, but still doesn't find books by category